eventkit_cloud/tasks/tests/test_export_tasks.py

```diff
@@ -351,31 +353,6 @@ def test_run_external_raster_service_export_task(self, mock_service, mock_reques
             external_raster_service_export_task.run(run_uid=self.run.uid, task_uid=str(saved_export_task.uid), stage_dir=stage_dir,
                                                     job_name=job_name)
 
-    @patch('eventkit_cloud.tasks.export_tasks.timezone')
```

Comment:
We do need a test for this `generate_qgs_style` function, but ok to leave it out for now.